style.css

```diff
@@ -1 +1,179 @@
 /*  Add your styling here */
+@media(max-width: 1024px){
+     
+  .upper-navigation{
+    justify-content: center;
+    font-size: 1.2rem;
+    margin-top: 25px; 
+    -webkit-font-smoothing: antialiased;
+  }
+  
+  .upper-navigation li {
+    display: inline-block;
+    margin-left: 75px;
+  }
+
+
+    
+    
+  
+}
+
+body{
+    background-color: pink;
+}
+
+.upper-navigation {
+    justify-content: space-around;
+    background-color: rgb(217, 236, 252);
+    position: relative;
+    margin: 0 auto;
+    
+  }
+  
+  .upper-navigation {
+    width: 970px;
+    height: 4.375rem;
+    margin: 0 auto;
+    color: #f721da;
+    justify-content: center;
+   
+  }
+  
+  .upper-navigation{
```

Comment:
this is now the third instance of this css class - choose one definition and keep it, remember that css is read from top to bottom, so only the last definition of the css class will be applied